rohmu/typing.py

```diff
@@ -25,15 +14,7 @@
 
 AnyPath = Union[str, bytes, "PathLike[str]", "PathLike[bytes]"]
 
-BinaryData = Union[
-    bytes,
-    bytearray,
-    memoryview,
-    "array[Any]",
-    mmap.mmap,
-    "ctypes._CData",
-    PickleBuffer,
-]
+BinaryData = Buffer
```

Comment:
Weird since we had mypy set to python3.9 since well before, but for some reason it now complains about overriding the type in FileWrap if we don't do this...